libraries/heft-config-file/src/ConfigurationFile.ts

```diff
@@ -132,8 +144,10 @@ export interface IOriginalValueOptions<TParentProperty> {
  */
 export class ConfigurationFile<TConfigurationFile> {
   private readonly _schemaPath: string;
+  private readonly _projectRelativeFilePath: string;
   private readonly _jsonPathMetadata: IJsonPathsMetadata;
   private readonly _propertyInheritanceTypes: IPropertyInheritanceTypes<TConfigurationFile>;
+  private readonly _rigsEnabled: boolean;
```

Comment:
This name should  match whatever name we choose for `IConfigurationFileOptions.disableRigs`

See https://github.com/microsoft/rushstack/pull/2231/files#r495503895